### ISSUE TITLE
Remove deprecated IUDPConnectedTransport

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4077]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#4077|
|Type|task|
|Created|2009-11-10 18:54:37Z|

```IUDPConnectedTransport``` in [twisted/internet/interfaces.py](twisted/internet/interfaces.py) is marked as deprecated so time to get rid of it.

Attachments:

* [4077.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/f91/f9100ae90639e687ef86741c13ac7713b84b8857/b5cc64315152ce97630021c76fb21d2be07e3cf2.diff) (961 bytes) - added by cyli on 2010-06-06 08:09:35Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__4077 4077
type__task task
reporter__thijs thijs
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy easy
time__1257879277000000 1257879277000000
changetime__1276052784000000 1276052784000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_1"></a>|@glyph set owner to @thijstriemstra|
|-|-|

OK!

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_2"></a>|@cyli removed owner|
|-|-|

Looks like it's been deprecated since 2008.  Nothing in trunk seems to implement it, and the full twisted test suite passes without it.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set owner to @cyli|
|-|-|

I don't like seeing years in .removal files.  Version numbers are much more helpful.  As far as I can tell, the deprecation was added in r24916 and first appeared in Twisted 9.0.  Can you change the .removal to say Twisted Twisted 9.0?  Otherwise looks great, please apply.

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_4"></a>|@cyli set status to `closed`|
|-|-|

(In [29270]) Applied patch 4077.diff from ticket 4077: remove deprecated IUDPConnectedTransport

Author: cyli
Reviewer: exarkun
Fixes: [#4077](https://github.com/twisted/twisted/issues/4077)

Removed deprecated IUDPConnectedTransport

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
